### ISSUE TITLE
Evaluate the state of a11y features

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/reactants-products-and-leftovers#76]
In a Slack DM, I asked @jessegreenberg and @zepumph:

RPAL is potentially going to resume work, adding PhET-iO and dynamic locale.

I noticed that you added keyboard navigation in https://github.com/phetsims/reactants-products-and-leftovers/issues/55.  But keyboard navigation does not work for me when running the sim.

It also looks like some description work was also done at the ScreenView level, see https://github.com/phetsims/joist/issues/355#issuecomment-338349680, but I don’t see those descriptions in master.

I haven’t been able to piece together what you were doing based on the GitHub issues and commits. Can you tell me more about what you did, and why, so that I can evaluate what needs to be done?

**Comment by jessegreenberg** [comment]
I reviewed the history. It looks like changes were small and then we decided not to work on this sim. Changes happened between 9/21/2017 and 10/20/2017. "a11y" commits after this window are related to regenerating the "a11y-view" in all sims (nothing sim specific).

Relevant changes I saw were:
- Some alt input related code (setting traversal order or focus management):
https://github.com/phetsims/reactants-products-and-leftovers/commit/42bf6c9a63a458f2faa7f94e6247f0268b631eb5
https://github.com/phetsims/reactants-products-and-leftovers/commit/f14435c76597af9b3a4e277dd4451fead264be09
https://github.com/phetsims/reactants-products-and-leftovers/commit/f95309876ebc7febe343e35f45811374dfbfde44

- Some Interactive Description additions:
~~https://github.com/phetsims/reactants-products-and-leftovers/commit/b27c5e6d127c3f42f261b73b5f490c7f758f09cc~~ These changes have been removed.

RPAL is no longer listed in perennial/data/interactive-description. The traversal order/focus management code will have no effect and can be removed if you prefer.

This would still be an easy sim to add alt-input to if PhET is interested. Work will be related to traversal order and focus management while changing game screens and as buttons pop in and out. Its possible a developer could quickly make good decisions about these without needing to involve design team (not to exclude, but take less design team and overhead time).

**Comment by pixelzoom** [comment]
I'm guessing that the focus traversal isn't working because package.json does not contain `"supportsInteractiveDescription": true`.

**Comment by pixelzoom** [comment]
And it looks like `this.accessibleOrder ` was changed to `this.pdomOrder` throughout.

**Comment by pixelzoom** [comment]
@jessegreenberg said:
> RPAL is no longer listed in perennial/data/interactive-description. The traversal order/focus management code will have no effect and can be removed if you prefer.

@jessegreenberg @zepumph @kathy-phet FYI...  I totally removed focus traversal code in the above commit.  I tried to work around it while doing TypeScript conversion #80 and PhET-iO instrumentation #78.  But it was getting in my way, and slowing me down.  Some of that code had TypeScript errors.  And (as is typical with a retrofit) PhET-iO instrumentation requires major changes to this sim.  So when this sim does get "alternative input" someday, I think it's best if we start with a clean slate.

Closing.
